lib/Dancer2/Core/App.pm

```diff
@@ -1111,6 +1111,11 @@ sub BUILD {
     $self->_init_hooks();
 }
 
+after BUILD => sub {
+    my $self = shift;
+    $self->log(core => 'Built config from files: ' . join(' ', @{$self->config_files}));
```

Comment:
Why have this inside an `after` wrapper, rather then inside the build method itself ?

Reply:
My thought was to make sure that everything (config, logger) was built. But you're right, it's not necessary. Updating.